runtime-image/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use the base App Engine Docker image, based on debian jessie.
-FROM gcr.io/google-appengine/debian8
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4:latest
```

Comment:
@nkubala With updating to Ubuntu, the structure tests license check gives the error:
```
=== RUN: [License Test]
--- FAIL
Error: Error reading license file for /usr/share/doc/openssl/copyright: Error retrieving file from container: API error (404): {"message":"Could not find the file ../libssl1.0.0/copyright in container 1bf212f5731aadcd07af70ce3b0f15498e1f088a4d8f0158e8f71b3ce7db0c3c"}
```
At first glance, this looks like it could be an issue with the structure tests.  When I look at the image, the `/usr/share/doc/openssl/copyright` file is a symbolic link that points to `../libssl1.0.0/copyright`.  That link exists and does contain a license.

Reply:
@sharifelgamal would know better than me, but I think you need to change the `debian` flag to `false` in your test definition

https://github.com/GoogleCloudPlatform/nodejs-docker/blob/master/runtime-image/test/test_config.yaml#L96

Reply:
Yes, I wasn't sure if it should be `true` or `false`.  I suspected `true` since Ubuntu is Debian based, but I'll try setting it to `false`.

Reply:
Yeah, setting that flag to false is the way to go for now.

Reply:
Yes, it looks like setting it to false fixes the problem, but it is good to know that the change is valid too.

Reply:
@sharifelgamal I've set the flag to false.  Do you see anything I should change with this PR or can I get a LGTM.  Thanks.